packages/xo-web/src/xo-app/logs/backup-ng/log-alert-header.js

```diff
@@ -58,6 +58,11 @@ export default decorate([
         await runBackupNgJob({
           id,
           schedule,
+          settings: {
+            '': {
+              bypassVdiChainsCheck: true,
```

Comment:
Bypass only when the failure is due to an un-healthy VDI chain.

Reply:
The spec says that we must bypass vdi chains check on every manual backup.
@julien-f, am I right ?

Reply:
Not at all, it only says that there must be a manual way to bypass the VDI chains check.